README.md

```diff
@@ -12,6 +12,20 @@ This will do the following:
 
 ## Usage
 
+### Prerequisites
+
+1. Obtain a [GitHub personal access token][generate-token].
+2. Make sure the [token is available as the `GITHUB_AUTH` environment variable][export-token].
+  For instance:
+  ```bash
+  export GITHUB_AUTH=abc123def456
+  ```
```

Comment:
Should we recommend a specific env setup here? Personally I use `direnv`, but I'm a bit worried that some nasty package in `node_modules` may sniff my env vars. I wonder, how have you configured it?

Reply:
I wouldn't.  Setting environment variables is a common enough task that it's a reasonable context to assume and I don't think this repo should be opinionated one way or another on how they're set.